dist/obs-server.spec

```diff
@@ -523,6 +523,7 @@ touch /srv/www/obs/api/log/production.log
 chown %{apache_user}:%{apache_group} /srv/www/obs/api/log/production.log
 
 %restart_on_update memcached
+touch /srv/www/obs/api/last_deploy
```

Comment:
please avoid the problems that adrian reverted it for - and catch errors. E.g. add a || true

Reply:
and add a comment that the code checks the mtime of the install time, so it's a little tougher to revert :)

Reply:
> please avoid the problems that adrian reverted it for - and catch errors

Hm, I think the problem why @adrianschroeter reverted it was that there was no last_deploy file when there is no .obsinfo. However, we always have now a last_deploy file during the build, so the file requires can not break the build anymore.

The other issue was that booting of the rails app failed when there is no last_deploy file. This fix is still in place.

But I can add the ``|| true`` but I'm not sure what it should fix. I will add a comment :+1: